relay-server/src/actors/events.rs

```diff
@@ -1497,7 +1498,10 @@ impl Handler<HandleEnvelope> for EventManager {
 
                 let checked = response.result.map_err(ProcessingError::EventRejected)?;
                 match checked.envelope {
-                    Some(envelope) => Ok(envelope),
+                    Some(envelope) => {
+                        envelope_summary.replace(EnvelopeSummary::compute(&envelope));
+                        Ok(envelope)
+                    }
```

Comment:
Is this the latest place where we need to update the envelope summary?

(I love how it can be written like this in the Rust syntax, btw.)

Reply:
Similar to the case in `endpoints/common.rs`, you'll have to make another update after calling `ProcessEnvelope` below once rate limiting can emit outcomes. For now, a comment will suffice. 

Technically, that would also apply to the `replace` you added here. However, it is fine to do it in this instance. Prior to invoking the `HandleEnvelope` handler, we split envelopes into event-related and event-unrelated items.